src/main/java/com/faforever/client/chat/ChatUserService.java

```diff
@@ -128,6 +130,15 @@ public void onChatUserGameChange(ChatUserGameChangeEvent event) {
       chatChannelUser.setStatusImage(null);
     }
   }
+
+  private String getLocalizedStatusGame(PlayerStatus status) {
+    return switch (status) {
```

Comment:
This would be better if the i18n key was an instance variable of the enum. See RatingMode.java for an example.

Reply:
Okay, I can do it better, but what about a separate `IDLE` variable that does not have a string? I do not want to put an empty string `''` or `null` for this because a man will have to write extra checks on an empty string or null before getting a localized string else we will get exception `NoSuchMessageException`.

How about adding this check to the `get(String, Object...)` method of `I18n` class? Like:
```java
public String get(String key, Object... args) {
    return key == null || key.isEmpty() ? null : get(userSpecificLocale, key, args);
  }
```

Or to implement a method like:
```java
public String tryGet(String key, Object... args) {
    try {
      return get(key, args);
    } catch (NoSuchMessageException e) {
      return null;
    }
  }
```

Reply:
I think it would be best to just give idle a message that way if it is used in the future it can be as well.